src/structures/Command.js

```diff
@@ -28,7 +30,9 @@ class Command
     //required
     this.name = settings.name;
     this.alias = settings.alias;
-    this.description = settings.description;
+    this.blurb = settings.blurb; 
+    this.longDescription = settings.longDescription; 
+    this.usages = settings.usages; 
```

Comment:
since not all commands have usages or longDescription, I'd try maybe something like `this.usages = settings.usages || []` to at least have it as a default instead of undefined